utils/scripts/generators/repository-generator.pl

```diff
@@ -348,6 +348,10 @@
             $all_entries      .= sprintf("\t\t\te.%-${longest_column_length}s = row[%s] ? strtod(row[%s], nullptr) : %s;\n", $column_name_formatted, $index, $index, $default_value);
             $find_one_entries .= sprintf("\t\t\te.%-${longest_column_length}s = row[%s] ? strtod(row[%s], nullptr) : %s;\n", $column_name_formatted, $index, $index, $default_value);
         }
+        elsif ($data_type =~ /timestamp/i) {
+            $all_entries      .= sprintf("\t\t\te.%-${longest_column_length}s = row[%s] ? reinterpret_cast<time_t>(row[%s]) : %s;\n", $column_name_formatted, $index, $index, $default_value);
+            $find_one_entries .= sprintf("\t\t\te.%-${longest_column_length}s = row[%s] ? reinterpret_cast<time_t>(row[%s]) : %s;\n", $column_name_formatted, $index, $index, $default_value);
+        }
```

Comment:
We don't really support timestamp (we support datetime) but this is only supporting one of the pieces in the generator. Where is the time object handling on the struct type